lib/core/build/classes/datawrapper/ChartQuery.php

```diff
@@ -29,7 +29,17 @@ public function createEmptyChart($user) {
         if ($user->isLoggedIn()) {
             $chart->setAuthorId($user->getId());
             $org = $user->getCurrentOrganization();
-            if (!empty($org)) $chart->setOrganization($org);
+            if (!empty($org)) {
+                $chart->setOrganization($org);
+
+                $settings = $org->getSettings();
+                if (isset($settings["default"]["folder"])) {
```

Comment:
did you test that this doesn't throw an error if `settings.default` doesn't exist?

Reply:
yes, that's how php's `isset` works: http://sandbox.onlinephpfunctions.com/code/a1137a03e9952099e4f6851cd6661ecbdf81e327